### ISSUE TITLE
Accessibility: Missing form labels

### ISSUE THREAD

**Issue by sjaeger24** [remotelearner/moodle-mod_questionnaire#556]
There are some missing form labels (detected by the WAVE tool):

- "Preview": calendar, slider, unticked radio buttons 
- "View all responses" > "List of responses": check boxes, radio buttons, Rate (scale 1...5), Yes/No, slider
- "Non-respondents": check boxes
- Answering the questionnaire: unticked radio buttons, calendar, slider
